### PR TITLE
NAS-108990 / 12.0 / Handled job response case when state is ABORTED for disk import

### PR DESCRIPTION
**Pull request by ryousaf**
NAS-108990

### CODE REVIEW

File: src/app/pages/storage/import-disk/import-disk.component.ts [hunk 1]
```diff
@@ -180,8 +180,8 @@ export class ImportDiskComponent implements OnDestroy, Formconfiguration {
     this.dialogRef.componentInstance.submit();
     this.dialogRef.componentInstance.success.subscribe((job_res) => {
```
Comment (william-gr) [inline src/app/pages/storage/import-disk/import-disk.component.ts:181]: Why are we reaching this code if the job didn't succeed?

Comment (ryousaf) [reply src/app/pages/storage/import-disk/import-disk.component.ts:181]: From what I gather, ABORTED is a valid state for the job and not failure. Failure is when there's an error in job execution. ABORTED is a valid state where no error occurred. We could improve the code by make a separate clearer indication like `dialogRef.componentInstance.success` (maybe `dialogRef.componentInstance.aborted`)

Comment (william-gr) [reply src/app/pages/storage/import-disk/import-disk.component.ts:181]: Yes, lets do that, please. Treating ABORTED as success would be wrong.

Comment (ryousaf) [reply src/app/pages/storage/import-disk/import-disk.component.ts:181]: The option to abort isn't provided in the UI, however. At least, not in this case. This case occurs when the job is initiated from the UI but aborted from the machine shell directly.

Comment (william-gr) [reply src/app/pages/storage/import-disk/import-disk.component.ts:181]: Right, but soon enough we will let it be done via UI

File: src/app/pages/storage/import-disk/import-disk.component.ts [hunk 2]
```diff
@@ -181,7 +181,7 @@ export class ImportDiskComponent implements OnDestroy, Formconfiguration {
...
       this.dialogRef.close();
       this.entityForm.success = true;
-      this.job.showLogs(job_res, T('Disk Imported: Log Summary'), T('Close'));
+      this.job.showLogs(job_res, T('Disk Imported: Log Summary')), T('Close');
```
Comment (william-gr) [inline src/app/pages/storage/import-disk/import-disk.component.ts:184]: This seems wrong?

Comment (ryousaf) [reply src/app/pages/storage/import-disk/import-disk.component.ts]: Yes. Misplaced parenthesis. Fixed.

**Review by william-gr — APPROVED** [review]

**Pull request merged**